### DISCUSSION TITLE
Add option for express checkout

### DISCUSSION THREAD

**Discussion by Liamm3** [craftcms/commerce-stripe#286] [Ideas]
Hello!

It would be nice to have a possibility to use the express checkout from Stripe: https://stripe.com/docs/elements/express-checkout-element.
![area_010](https://github.com/craftcms/commerce-stripe/assets/29706892/53c3c048-d8fe-4c2a-afcd-8806b85f806c)

Maybe I am mistaken, but I think there is not an easy solution to this at the moment. Feel free to correct me if I'm wrong. :)

**Comment by philipboomy** [comment]
Very related so will post here:

It would be nice to support 1 step Apple/Google Pay on product pages similar to what Shopify does here: https://dowsedesign.co.uk/collections/accessories/products/striped-ona-soft-bag

We might be able to use this one here https://docs.stripe.com/elements/express-checkout-element 

We believe this will greatly increase conversion rates.

**Reply by craftunit-liam** [reply]
Hey there! We released a plugin for implementing the Stripe Express Checkout Element: https://plugins.craftcms.com/stripe-express-checkout?craft5 Might be worth checking out. :)
